examples/qml_minimal/CMakeLists.txt

```diff
@@ -91,7 +99,11 @@ target_link_libraries(${APP_NAME}_lib INTERFACE
 
 # ANCHOR: book_cmake_executable
 # Define the executable with the C++ source
-add_executable(${APP_NAME} cpp/main.cpp)
+if(BUILD_WASM)
+    qt_add_executable(${APP_NAME} cpp/main.cpp)
```

Comment:
would be good to have a comment as to why we need this (otherwise the html file is not generated). And would still be good if we could figure out how to make this work with `add_executable` so that it's an additional line for WASM instead of having an if block :thinking: